.vscode/tasks.json

```diff
@@ -0,0 +1,15 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
```

Comment:
comments are invalid in JSON. Also, this is a .vscode file. I assume this should actually be ignored?

Reply:
This is very useful when developing. If you don't use vscode, it won't harm you. I will remove the comments tho @chris-decker-volusion.